app/apptesting/gamm.go

```diff
@@ -58,6 +58,16 @@ func (s *KeeperTestHelper) PrepareBalancerPoolWithCoins(coins ...sdk.Coin) uint6
 	return s.PrepareBalancerPoolWithCoinsAndWeights(coins, weights)
 }
 
+// PrepareBalancerPoolWithCoinsAndSwapFee returns a balancer pool
+// consisted of given coins with equal weight set with provided swap fee.
+func (s *KeeperTestHelper) PrepareBalancerPoolWithCoinsAndSwapFee(swapFee sdk.Dec, coins ...sdk.Coin) uint64 {
```

Comment:
Note: planning to create an issue to streamline these helpers to reduce redundant boilerplate

Reply:
Yes, IMO we should just have a single helper that uses default params unless overridden by user input. Wondering if you see this the same way

Reply:
Tracking here: https://github.com/osmosis-labs/osmosis/issues/3675